Wootric/slate
# Add Flutter to docs

Add Flutter to menu

## Changes

- Update `doc_icons.png` (thanks @DanPitrowiski )
- Update `index.html` and css

## Test

1. Checkout this branch
2. Run server
3. Check flutter options appears and clicking on it takes you to https://github.com/Wootric/WootricSDK-flutter

## Jira

https://jira.inmoment.com/browse/DEV-26837
